modules/outputs.nix

```diff
@@ -111,7 +111,8 @@ in
       };
       uimage = liminix.builders.uimage {
         commandLine = concatStringsSep " " config.boot.commandLine;
-        inherit (config.hardware) loadAddress entryPoint;
+        inherit (config.boot) commandLineDtbNode;
+        inherit (config.hardware) loadAddress entryPoint alignment;
         inherit (config.boot) imageFormat;
```

Comment:
comment: At some future point we'll have to tidy up the 'boot' vs 'hardware' config space because it's a bit of a mess right now.